sycl/plugins/unified_runtime/ur/adapters/hip/event.cpp

```diff
@@ -310,7 +322,13 @@ UR_APIEXPORT ur_result_t UR_APICALL urEventGetNativeHandle(
 ///
 /// \return UR_RESULT_ERROR_UNSUPPORTED_FEATURE
```

Comment:
I missed that I should update this function descriptor. Will do this following review.